app/internal/wm/os_js.go

```diff
@@ -36,6 +36,9 @@ type window struct {
 	composing             bool
 	requestFocus          bool
 
+	chanAnimation chan bool
+	chanRedraw    chan bool
```

Comment:
I use `chan struct{}` everywhere else for notification channels. Please do that here as well.